entities.rs

```diff
@@ -0,0 +1,753 @@
+use alloc::vec::Vec;
```

Comment:
You seem to have added a copy of the entire file to the root.

Reply:
ups